solr/core/src/java/org/apache/solr/cli/Tool.java

```diff
@@ -52,5 +53,7 @@ default String getFooter() {
 
   List<Option> getOptions();
 
+  List<OptionGroup> getOptionGroups();
```

Comment:
I know this PR is a draft, but is it eventually better to go for `Options getOptions()` instead of `getOptions()` and `getOptionGroups()`? If a list is needed `Options#getOptions()` may be used instead.

`Options#addOptions(Options options)` does also exist if necessary.

Reply:
definitly!   I didn't realize....   Also, if you want to push up to it, please do!